src/include/sof/wait.h

```diff
@@ -145,6 +146,25 @@ static inline int wait_for_completion_timeout(completion_t *comp)
 	}
 }
 
+static inline int wait_for_completion_idelay(int (*condition)(void *data), void *data, int clk)
```

Comment:
int clk should be named delay_ms or delay_us depending on time measurement.

Reply:
yes, I will refine my patch